tests/conftest.py

```diff
@@ -180,6 +180,23 @@ async def tz_bot(timezone, bot_info):
         return default_bot
 
 
+@pytest.fixture(scope="session")
+async def tz_bots(tzinfos, bot_info):
```

Comment:
1) I recommend to add a comment in this fixture about why we use a list here instead of properly parametrizing (speed)
2) do we still need the `tz_bot` fixture for anything else, or can it be removed, then?
3) Can we make sure that the tests at least point out the timezone on failure?

Reply:
> do we still need the tz_bot fixture for anything else, or can it be removed, then?

we do still use it in the `NonRequest` tests, where the time penalty is very negligible.

> Can we make sure that the tests at least point out the timezone on failure?

that would be hard to do without making it even more complex, but let me have an another look again